daemon/create_test.go

```diff
@@ -0,0 +1,69 @@
+package daemon
+
+import (
+	"testing"
+
+	"github.com/docker/docker/runconfig"
+)
+
+func TestValidateVolumePath(t *testing.T) {
+
+	config := &runconfig.Config{Volumes: make(map[string]struct{})}
+	hostConfig := &runconfig.HostConfig{}
+
+	// given invalid src
+	hostConfig.Binds = []string{"data:/data"}
+	if err := validateVolumePath(config, hostConfig); err == nil {
+		t.Fatal("Expected validateVolumePath error, got nil")
+	}
+
+	// given invalid dest
+	hostConfig.Binds = []string{"data:/data"}
```

Comment:
This looks exactly like the test case above. Maybe it was intended to be `[]string{"/data:data"}`?